reference-data/omod/src/main/java/org/bahmni/module/referencedata/labconcepts/model/event/SellableTypeEvent.java

```diff
@@ -0,0 +1,45 @@
+package org.bahmni.module.referencedata.labconcepts.model.event;
+
+import org.openmrs.Concept;
+import org.ict4h.atomfeed.server.service.Event;
+import org.joda.time.DateTime;
+import org.openmrs.ConceptAttribute;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.UUID;
+
+public class SellableTypeEvent implements ConceptServiceOperationEvent {
+
+    public static final String RESOURCE_TITLE = "reference data";
+    public static final String SELLABLE_ATTR_NAME = "sellable";
+    private final String url;
+    private final String category;
+    private List<String> supportedOperations = Arrays.asList("saveConcept", "updateConcept", "retireConcept", "purgeConcept");
+
+    public SellableTypeEvent(String url, String category) {
+        this.url = url;
+        this.category = category;
+    }
+
+    @Override
+    public Event asAtomFeedEvent(Object[] arguments) throws URISyntaxException {
+        Concept concept = (Concept) arguments[0];
+        String url = String.format(this.url, this.category, concept.getUuid());
```

Comment:
The URL is what is published here. Going by this line, the content in the event is going to be something like - "/openmrs/ws/rest/v1/reference-data/**sellable**/fe334cb7-t3tb-0037-70f7-kjditree2222"

We actually want the URL to be "/openmrs/ws/rest/v1/reference-data/**resources**/fe334cb7-t3tb-0037-70f7-kjditree2222", because thats what "ResourcesController" controller exposes API to.